homeassistant/components/sabnzbd.py

```diff
@@ -102,7 +102,7 @@
                                         hass.config.path(CONFIG_FILE))
         api_key = conf.get(base_url, {}).get(CONF_API_KEY, '')
 
-    sab_api = SabnzbdApi(base_url, api_key)
+    sab_api = SabnzbdApi(base_url, api_key, "sabnzbd")
```

Comment:
With the new version of the library we should also pass in the ClientSession to `SabnzbdApi` using https://github.com/home-assistant/home-assistant/blob/aeb21596a0acb032cf494893eb236bad44cdc006/homeassistant/helpers/aiohttp_client.py#L26